layouts/community/ergodox/nstickney/keymap.c

```diff
@@ -0,0 +1,217 @@
+#include QMK_KEYBOARD_H
+
+/*
+ * This keymap simulates the key limitations of an Iris on an ErgoDox. See the
+ * matching iris layout (/keyboards/iris/keymaps/nstickney) for further
+ * information.
+ */
+
+// Layers
+#define BASE 0 // Base layer
+#define SYMB 1 // Symbols
+#define SYSH 2 // Symbols, shifted
+#define NUMP 4 // Numpad
+#define FCTN 8 // Function
+
+// Tap Dancing
+void dance_lock (qk_tap_dance_state_t *state, void *user_data) {
+	switch (state->count){
+		case 1: // Press once for LGUI
+			register_code(KC_LGUI);
```

Comment:
You could actually use `tap_code` for all of these.  It does the same thing, but with an optional delay.

Reply:
https://docs.qmk.fm/#/feature_macros?id=tap_codeltkcgt

Reply:
:+1: